src/docfx/restore/RestoreGit.cs

```diff
@@ -93,8 +93,11 @@ async Task FetchOrCloneRepo()
             {
                 if (GitUtility.IsRepo(restoreDir))
                 {
+                    // turn on core.longpaths before fetch
+                    await GitUtility.SetCoreLongPaths(restorePath, true);
```

Comment:
>SetCoreLongPaths [](start = 37, length = 16)

if the repo is cloned using `-c core.longpaths=true`, do we still need to configure it on fetch?

Reply:
If the repo is migrated from a `core.longpaths=false` state, it needs to be switched on. 
Do you recommend check `core.longpaths` before fetch and decide if it needs to be turned on?

Reply:
no, I was thinking we may safely assume this repo is totally managed by *us*, so it won't have a `core.longpaths=false` state given we haven't shipped v3 yet.

Reply:
for migration? some repo were already cloned by us using core.longpaths=false